cmd/crio/wipe.go

```diff
@@ -67,6 +67,14 @@ func crioWipe(c *cli.Context) error {
 		return handleCleanShutdown(config, store)
 	}
 
+	// If we make it here, and we're set to wipe internally
+	// crio wipe is done, unless force is set.
```

Comment:
Is there some punctuation missing between the first and the second line? It's a bit hard to parse this comment.